packages/protocol/src/protocol.yml

```diff
@@ -1611,7 +1590,7 @@ Page:
     goBack:
       title: Go back
       parameters:
-        timeout: number?
+        timeout: number
```

Comment:
I'm a little worried that it's very easy to break in the ports without noticing. Maybe we can store per-client default timeouts on the server end?

Reply:
I'll try to roll into languages and see how it goes.